### PR TITLE
Project Finished

### PR DESCRIPTION
**Pull request by mgmediaweb**
# Features

- The theme of my project is a movie WebApp, and the information is provided by the IMDB API.
- I use React-Create-App, and Redux-Toolkit.
- I use 3 connection APIs for Theatres Movies, Top Movies, and Top Stars. all of them are stored in the global Store.
- I make a horizontal slider component to show the Theatres and Top Movies.
- I make a specific horizontal sliders component for the actors, this component I re-use on the homepage and detail page (with the personage name in the movie).
- I Create a detailed page with the all information of the store and plus a connection to 2 APIs more, to show the trailer of the movie and the top 20 casts of the actors.
- All units and integrations tests passed.
- All Linters passed.

## Live View 
https://mv-imdb.herokuapp.com/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,12 +56,6 @@ To view this document on your local computer:
...
 
-👤 **Waqas Ahmad**
-
-- GitHub: [@githubhandle](https://github.com/waqaskanju)
-- Twitter: [@twitterhandle](https://twitter.com/waqaskanju)
-- LinkedIn: [@LinkedIn](https://www.linkedin.com/in/waqaskanju/)
-
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
```
Comment (BenMukebo) [inline README.md:61]: - [x] According to the project requirements, the README file should contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly add this ✍🏻

**Review by BenMukebo — CHANGES_REQUESTED** [review]
## Hi @Lordkaito, 

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


### To Highlight 🎉

- You have followed  JavaScript best practices. ✔️ 
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️
- Good presentation video. ✔️ 
- Your app has been well-deployed to Heroku. ✔️ 
- You have a basic project structure ✔️ 
- App has unit tests for several components of the app ✔️ 
- Your Design looks awesome, keep rocking :rocket: 

## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (mgmediaweb) [reply README.md:61]: I applied your comment, thank you for your support

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @mgmediaweb 

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**